lib/src/models/track.dart

```diff
@@ -2,11 +2,15 @@ class Track {
   int id;
   int participantId;
   int? incomingTrackId;
+  TrackKind? kind;
 
   Track.fromJson(Map<String, dynamic> json)
       : id = int.parse(json['id']),
         participantId = json['participantId'],
-        incomingTrackId = int.tryParse(json['incomingTrackId'] ?? '');
+        incomingTrackId = int.tryParse(json['incomingTrackId'] ?? ''),
+        kind = json['audio'] ?? false ? TrackKind.audio : (json['video'] ?? false ? TrackKind.video : null);
```

Comment:
This line could benefit from adding a comment. Also, this line seems longer than 80 chars, please make sure your IDE is set up to limit each line to 80 chars.